STT/Bindings/SttBaseCollectionViewSource.swift

```diff
@@ -220,6 +220,12 @@ open class SttBaseCollectionViewSource<TPresenter: SttViewInjector>: NSObject, U
     }
     
     open func scrollViewDidEndDecelerating(_ scrollView: UIScrollView) { }
+    
+    open func scrollViewDidEndDragging(_ scrollView: UIScrollView, willDecelerate decelerate: Bool) { }
```

Comment:
think probably it would be better create next inheritance
BaseScrollSource { }
BaseTableViewSource: BaseScrollSource { }
BaseCollectionViewSource: BaseScrollSource { }
it take from you about 15m, Pls do it if you start implementating this method